pkg/report/guilty.go

```diff
@@ -52,3 +56,24 @@ nextFile:
 	}
 	return ""
 }
+
+func GetMaintainers(linux string, file string) ([]string, error) {
+	// ./scripts/get_maintainer.pl is a Linux kernel script.
+	output, err := osutil.RunCmd(time.Minute, linux, "./scripts/get_maintainer.pl", "-f", file)
+	if err != nil {
+		return nil, fmt.Errorf("get_maintainer.pl failed: %v\n%v", err, string(output))
+	}
+	parts := strings.Split(string(output), "\n")
+	var mtrs []string
+	for _, part := range parts {
+		if !strings.Contains(part, "maintainer") && !strings.Contains(part, "supporter") &&
+			!strings.Contains(part, "open list") && !strings.Contains(part, "commit_signer") {
+			continue
+		}
+		email := regexp.MustCompile(`[a-zA-Z\.\-]+@[a-zA-Z\.\-]+`).FindString(part)
```

Comment:
Please don't reimplement whole RFC5322. The first part obviously can contain numbers as well. There is `mail.ParseAddress`:
https://golang.org/pkg/net/mail/#ParseAddress